e2e/cypress/support/index.js

```diff
@@ -41,14 +42,28 @@ Cypress.on('window:alert', (text) => {
 });
 
 before(function () {
-  const autoLogin = function () {
-    return cy.loginViaForm();
-  };
-  autoLogin().then((msg) => {
-    cy.log(msg);
+  cy.clearLocalStorageSnapshot();
+
+  cy.loginViaAPI().then(() => {
```

Comment:
this loginViaAPI was causing tests to fail randomly. This is the reason I switched to loginViaForm. Have you tested this roughly (like running the entire e2e suite 4 times and don't have any test failed as a consequence of login) ?

Reply:
I've tried loginViaForm and it's also flaky as it's always about how fast the test is running, how fast the API is responding and the timeouts we set. And API authentication is way more reliable than going through the form.